index.html

```diff
@@ -2308,7 +2308,7 @@ <h3>Context Validation</h3>
 executed business rules specific to the input in the document. For more
 rationale related to this algorithm, see Section [[[#validating-contexts]]].
 This algorithm takes inputs of a document ([=map=] |inputDocument|), a set of
-approved JSON-LD Contexts ([=map=] |expectedContext|), and a boolean to
+known JSON-LD Contexts ([=array=] |knownContext|), and a boolean to
```

Comment:
There are two instances of `expectedContext` in the text below that need to be updated to `knownContext`.

Reply:
Whoops! Done in https://github.com/w3c/vc-data-integrity/pull/302/commits/93821e31d6b001ed113a3bcdb21a02241b399198